src/MernelPlatform/ByteOrderStream.hpp

```diff
@@ -11,6 +11,16 @@
 #include <deque>
 #include <map>
 #include <type_traits>
+#include <limits>
+
+/// Conflicts with "endian.h" in unix
+#ifdef BIG_ENDIAN
```

Comment:
No, that's not good, I'd better rename that field insted. wait a minute.

Reply:
Done. Can you rebase please?

Reply:
Yeah, rename is a better choice.